TwitchDownloaderCLI/Modes/RenderChat.cs

```diff
@@ -0,0 +1,93 @@
+﻿using SkiaSharp;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using TwitchDownloaderCLI.Tools;
+using TwitchDownloaderCore;
+using TwitchDownloaderCore.Options;
+using TwitchDownloaderCLI.Modes.Arguments;
+
+namespace TwitchDownloaderCLI.Modes
+{
+    internal class RenderChat
+    {
+        internal static void Render(ChatRenderArgs inputOptions)
+        {
+            ChatRenderOptions renderOptions = new()
+            {
+                InputFile = inputOptions.InputFile,
+                OutputFile = inputOptions.OutputFile,
+                BackgroundColor = SKColor.Parse(inputOptions.BackgroundColor),
+                MessageColor = SKColor.Parse(inputOptions.MessageColor),
+                ChatHeight = inputOptions.ChatHeight,
+                ChatWidth = inputOptions.ChatWidth,
+                BttvEmotes = (bool)inputOptions.BttvEmotes,
+                FfzEmotes = (bool)inputOptions.FfzEmotes,
+                StvEmotes = (bool)inputOptions.StvEmotes,
+                Outline = inputOptions.Outline,
+                OutlineSize = inputOptions.OutlineSize,
+                Font = inputOptions.Font,
+                FontSize = inputOptions.FontSize,
+                ChatBadgeMask = inputOptions.BadgeFilterMask,
+                MessageFontStyle = inputOptions.MessageFontStyle.ToLower() switch
+                {
+                    "normal" => SKFontStyle.Normal,
+                    "bold" => SKFontStyle.Bold,
+                    "italic" => SKFontStyle.Italic,
+                    "italics" => SKFontStyle.Italic,
+                    _ => throw new NotImplementedException("Invalid message font style")
+                },
+                UsernameFontStyle = inputOptions.UsernameFontStyle.ToLower() switch
+                {
+                    "normal" => SKFontStyle.Normal,
+                    "bold" => SKFontStyle.Bold,
+                    "italic" => SKFontStyle.Italic,
+                    "italics" => SKFontStyle.Italic,
+                    _ => throw new NotImplementedException("Invalid username font style")
```

Comment:
It would be nice to include what are the valid ones in the switch defaults.

Reply:
Good point